python/in3/exception.py

```diff
@@ -24,10 +24,10 @@ class HashFormatException(IN3BaseException):
 
 
 class ClientException(IN3BaseException):
```

Comment:
It is actually more standard to name exceptions `errors` in python.

So, all these should ideally by like `ClientError` etc.

However, that would be a substantial breaking change.

Reply:
Hi @unparalleled-js! @simon-jentzsch asked me to come and take a look into you considerations, thanks for contributing, we really appreciate it!

Reply:
We used exceptions here to signal the user of our library that it is not a python error but a library exception that was thrown. 

You are correct that the official standard is to call all exceptions `...Error`, as in the [official docs](https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions) they extend Error from the base Exception class. It is also pointless to debate if this custom makes sense or not.

We will consider bumping the version and changing it all to Error in the future, to match community customs. Thanks again!

Reply:
Thank you! I would like to work on this project more, particularly the python version. I find it pretty useful and am using it in one my own projects, so I am invested in its success. I am happy to make any adjustments, but this PR mainly is to address the error message such that is says `.eth` instead of `.ens` for domain formatting.  I know the custom exception I made is not really a user-facing one, meaning users will not need to catch it, but it is better to re-use the logic rather than fix the bug in several spots

Reply:
I am personally very glad to learn you are using it and invested in improving it! Count on me and our team at Blockchains to support you and fix the bugs you might encounter in the bound C library.